### DISCUSSION TITLE
release date

### DISCUSSION THREAD

**Discussion by Ap0ph1s69** [egzumer/uv-k5-firmware-custom#522] [General]
hell didn't know where to post this but was just wondering is there any more releases or is v22 the final build
thanks for all the hard work

**Comment by egzumer** [comment]
There is a release section in this repository, all releases are there.

**Reply by Ap0ph1s69** [reply]
what I meant to say is your firmware is on V0.22 will a v0.23 be released or is that the final build thanks

**Reply by egzumer** [reply]
I'm not working on it right now.
